README.md

```diff
@@ -40,24 +39,33 @@ Terraform-nomad-minio module is IaC - infrastructure as code. Module contains a
 ### Requirements
 
 #### Required modules
+No modules required.
 
 #### Required software
 - [GNU make](https://man7.org/linux/man-pages/man1/make.1.html)
-- [consul](https://releases.hashicorp.com/consul/) binary available on `PATH` on the local machine.
+- [Consul](https://releases.hashicorp.com/consul/) binary available on `PATH` on the local machine.
 
 #### Other
 
 ## Usage
+The following command will run an example with standalone instance of minio.
```

Comment:
```suggestion
The following command will run an example with standalone instance of Minio.
```

Maybe be consistent with uppercase. Should check the whole document for any missed namings.

Reply:
Actually, it was a discussion in mattermost the other day, where it was mentioned we should use "egennavn"! So it should be MinIO